src/models/contracts/objectExplorer/createSessionRequest.ts

```diff
@@ -0,0 +1,67 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import {NotificationType, RequestType} from 'vscode-languageclient';
+// tslint:disable-next-line:no-unused-variable
+import { ConnectionDetails } from '../connection';
```

Comment:
Why is this necessary?

Reply:
Linter otherwise complains with the following error - 
```
ERROR: (no-unused-variable) vscode-mssql/src/models/contracts/objectExplorer/createSessionRequest.ts[8, 1]: All imports on this line are unused.
```

Reply:
Which makes sense since it looks like you aren't actually using ConnectionDetails anywhere. So you should just remove it if that's the case.

Reply:
Oh wait nevermind it's in the generic definition. And actually I recall seeing that issue before...there was a way to fix it though IIRC. Let me look it up - otherwise this is fine